cmake/modules/AddSwift.cmake

```diff
@@ -1320,6 +1329,8 @@ function(add_swift_library name)
         # Configure variables for this subdirectory.
         set(VARIANT_SUFFIX "-${SWIFT_SDK_${sdk}_LIB_SUBDIR}-${arch}")
         set(VARIANT_NAME "${name}${VARIANT_SUFFIX}")
+        set(MODULE_VARIANT_SUFFIX "-swiftmodule-${SWIFT_SDK_${sdk}_LIB_SUBDIR}-${arch}")
```

Comment:
Do you mean the "${SWIFT_SDK_${sdk}_LIB_SUBDIR}-${arch}" part in MODULE_VARIANT_SUFFIX?  If so, then yeah, I'll fix that.